airflow/www/views.py

```diff
@@ -1893,6 +1910,83 @@ def gantt(self, session=None):
             root=root,
         )
 
+    @expose('/redirect')
+    @login_required
+    @wwwutils.action_logging
+    def redirect(self):
```

Comment:
Similar comment as above re: the use of the term "redirect" - this really just returns a list of URLs (or "extra_links", as they are called elsewhere.)